react-game/webpack.config.js

```diff
@@ -0,0 +1,61 @@
+const path = require('path')
+const webpack = require('webpack')
+const HTMLWebpackPlugin = require('html-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+
+module.exports = {
+    context: path.resolve(__dirname, 'src'),
+    mode: 'development',
+    entry:{
+        index: './index.tsx'
+    },
+    resolve: {
+        extensions: ['.tsx', '.ts', '.js'],
+    },
+    output: {
+        filename: '[name].bundle.js',
+        path: path.resolve(__dirname, 'dist')
+    },
+    devServer: {
+        historyApiFallback: true,
+        contentBase: path.resolve(__dirname, './dist'),
+        open: true,
+        compress: true,
+        hot: true,
+        port: 8080,
+    },
+    module: {
+        rules: [
+            //{ enforce: 'pre', test: /\.js$/, loader: "eslint-loader"},
+            {
+                test: /\.tsx?$/,
+                use: 'ts-loader',
+                exclude: /node_modules/,
+            },
+            {
+                test: /\.css$/, 
+                use: [MiniCssExtractPlugin.loader, 'css-loader']
+            },
+            {
+                test: /\.(svg|png|jpe?g|gif|mp3|m4a)$/i,
+                use: [
+                    {
+                    loader: 'file-loader?name=assets/[name].[ext]',
+                    },
+               ],
+            },
+        ]
+    },
+    plugins: [
+        new HTMLWebpackPlugin({
+            template: './index.html'
+        }),
+        new MiniCssExtractPlugin({
+            filename: '[name].bundle.css'
+        }),
+        new webpack.HotModuleReplacementPlugin(),
+        new CleanWebpackPlugin()
+    ]
+};
+

```

Comment:
currently the project structure (folders and files) is: 
react-game/
  |__src/
  |__package.json
  |__etc.
Readme.md

My recommendation is to bring everything up a level, so that src , package.json etc are at the root
src/
package.json
etc.
Readme.md